### PR TITLE
Add the new release Apple device identifier

### PR DESCRIPTION
**Pull request by summic**
## What's new in this PR?

Add the new release Apple device identifier

iPhone 12 and  the new iPad

### CODE REVIEW

**Comment by zenkins** [comment]
Can one of the admins verify this patch?

**Comment by billypchan** [comment]
Thx @summic for your contribution! We will review this PR and test it.

File: Source/Utility/Device.swift [hunk 1]
```diff
@@ -74,7 +79,15 @@ import Foundation
...
         case "iPad11,3", "iPad11,4":                    return "iPad Air 3"
+        case "iPad11,6":                                return "iPad 8th Gen (WiFi)"
+        case "iPad11,7":                                return "iPad 8th Gen (WiFi+Cellular)"
+        case "iPad13,1":                                return "iPad air 4th Gen (WiFi)"
+        case "iPad13,2":                                return "iPad air 4th Gen (WiFi+Cellular)"
+        case "iPad13,4","iPad13,5","iPad13,6", "iPad13,7":  return "iPad Pro 11 inch 3rd Gen"
+        case "iPad13,8","iPad13,9","iPad13,10","iPad13,11": return "iPad Pro 12.9 inch 5th Gen"
         case "AppleTV5,3":                              return "Apple TV"
+        case "AppleTV6,2":                              return "Apple TV 4K"
+        case "AppleTV11,1":                             return "Apple TV 4K Apple TV 4K (2nd generation)"
```
Comment (billypchan) [inline Source/Utility/Device.swift:90]: we do not support Apple TV so these are not needed

Comment (summic) [reply Source/Utility/Device.swift]: Sorry for submitting the wrong code, I'll delete the pr and start over

**Pull request closed without merging**